full_cluster_replacement_test.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from longevity_test import LongevityTest
+from sdcm import prometheus
+from sdcm.sct_events.system import InfoEvent
+from sdcm.utils import loader_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+class FullClusterReplacement(LongevityTest, loader_utils.LoaderUtilsMixin):
+    """
+    The purpose of these tests is replacement of all nodes in the ScyllaDB.
+    Cluster Replacement QA task: https://github.com/scylladb/qa-tasks/issues/1164
+    Cluster replacement test plan: https://docs.google.com/document/d/1aKCTs6SiH9ORDYvkAQ8upOpVwZqaxsW9VfbJqKG63g0/edit
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.metrics_srv = prometheus.nemesis_metrics_obj()
+        self.stress_queue = []
+
+    def replace_node(self, node_to_be_removed, dc_idx=1):
```

Comment:
Please change the name of it to be "replace_node_by_add_and_decommission"

Reply:
ok

Reply:
Done